src/tests/components/ErrorBoundary.test.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import { render } from '@testing-library/react'
+import { ErrorBoundary } from '../../components/UI'
+
+const ErrorText = 'Error message'
+const ErrorChild = <div className='custom-class'>{ErrorText}</div>
+
+describe(`ErrorBoundary component test`, () => {
```

Comment:
Rename: `Component:ErrorBoundary`. Then we can use the same pattern with other tests, like:

`Service:AuthService`
`Utilities:Validation`

Reply:
I will change it after merge because there is more file which needs to be changed